PlayStore/tr-TR.xml

```diff
@@ -6,62 +6,55 @@ Instructions for manual translation:
 - Translate the properties making sure not to overpass the maximum character number.
 - Currently this file is copied manually, line breaks are kept and no need to scape special characters.
 
-If you added/modified the following file add your name here to give credit:
-Yusuf Emen
-
+If you added/modified the following file add your name here to give credit: Hüseyin Filiz (HuseyinFiliz.com.tr) & Yusuf Emen
+ 
 
  -->
 <parameters>
-    <title max_chars="30">Tıkla Sohbet Et</title>
-    <description_short max_chars="80">Herhangi bir numarayla bir WhatsApp sohbeti başlatın.</description_short>
-    <description_long max_chars="4000">
-Sorumluluk reddi: Bu uygulamanın WhatsApp Inc. ile bir bağlantısı yoktur.
-Previous name: "WhatsApp'ta Aç (tıkla Sohbet Et) [small, no ads]", changed to comply with WhatsApp and Google guidelines.
-------------------------------------------------------------------------------------------
 
-Bu uygulama, girdiğiniz herhangi bir numara ile sohbet başlatmak için (Siz ve girdiğiniz numara arasında, diğer sohbetleri görmek mümkün DEĞİLDİR.) WhatsApp'ın herkese açık 'tıkla sohbet et' (Daha fazla bilgi için: https://faq.whatsapp.com/tr/android/26000030/) özelliğini kullanır.
-Rehberinizde kişi oluşturulmaz. Sohbet başlatmak için numarayı rehbere kaydetmekle uğraşmazsınız. Yalnızca uygulamayı açın, numarayı girin, butona basın ve sohbet açılsın. (Numara mevcutsa, aksi takdirde WhatsApp bunu belirtecektir.)
+    <!-- Preferably use the name from https://faq.whatsapp.com/general/chats/how-to-use-click-to-chat/ in your language -->
+    <title max_chars="30">Tıkla Sohbet Et</title>
 
-Özellikle şu gibi durumlarda işe yarar:
-- Bilinmeyen bir numara sizi aradı ve siz de bu numaranın WhatsApp'ı olup olmadığını mı merak ediyorsunuz?
-- Birine mesaj göndermek istiyorsunuz ama numarasını rehbere kaydetmek istemiyor musunuz?
-- Kendinizle nr sohbet başlatmak mı istiyorsunuz?
-Sadece uygulamayı açın ve numarayı girin. (Ülke konuyla birlikte.)
+    <description_short max_chars="80">Rehbere kaydetmeden bir numarayla WhatsApp'tan konuşmanızı sağlayan küçük ve reklamsız bir uygulama.</description_short>
```

Comment:
This description_short is longer than 80 characters. Can you reduce it? It can't be updated if longer.